docs/build-instructions/windows.md

```diff
@@ -2,40 +2,36 @@
 
 ## Requirements
 
-### On Windows 7
-  * [Visual C++ 2010 Express](http://www.visualstudio.com/en-us/downloads/download-visual-studio-vs#DownloadFamilies_4)
-  * [Visual Studio 2010 Service Pack 1](http://www.microsoft.com/en-us/download/details.aspx?id=23691)
-  * [Node.js](http://nodejs.org/download/) (0.10.x or above)
-    * For 64-bit builds of node and native modules you **must** have the
-    [Windows 7 64-bit SDK](http://www.microsoft.com/en-us/download/details.aspx?id=8279).
-    You may also need the [compiler update for the Windows SDK 7.1](http://www.microsoft.com/en-us/download/details.aspx?id=4422)
-  * [Python](https://www.python.org/downloads/) v2.7.
+### General
+  * [Node.js](http://nodejs.org/en/download/) (0.10.x or above)
+  * [Python](https://www.python.org/downloads/) v2.7.x
     * The python.exe must be available at `%SystemDrive%\Python27\python.exe`.
       If it is installed elsewhere, you can create a symbolic link to the
       directory containing the python.exe using:
       `mklink /d %SystemDrive%\Python27 D:\elsewhere\Python27`
   * [GitHub Desktop](http://desktop.github.com/)
 
+### On Windows 7
+  * [Visual C++ 2010 Express](http://www.visualstudio.com/en-us/downloads/download-visual-studio-vs#DownloadFamilies_4)
+  * [Visual Studio 2010 Service Pack 1](http://www.microsoft.com/en-us/download/details.aspx?id=23691)
+  * [Windows 7 64-bit SDK](http://www.microsoft.com/en-us/download/details.aspx?id=8279) for 64-bit builds of Node and native modules
+    * You may also need the [compiler update for the Windows SDK 7.1](http://www.microsoft.com/en-us/download/details.aspx?id=4422)
+
 ### On Windows 8 or 10
   * [Visual Studio Express 2013 or 2015 for Windows Desktop](http://www.visualstudio.com/en-us/downloads/download-visual-studio-vs#DownloadFamilies_2)
-    * For VS 2015, be sure to customize the installation to include Visual C++. It's not installed by default.
-    * Some have experienced issues with Node locating C++ on VS 2015. If so, try VS 2013.
-  * [Node.js](http://nodejs.org/download/) (0.10.x or above)
-  * [Python](https://www.python.org/downloads/) v2.7.x (required by [node-gyp](https://github.com/TooTallNate/node-gyp))
-  * [GitHub Desktop](http://desktop.github.com/)
+    * To ensure that node-gyp knows what version of Visual Studio is installed, set the `GYP_MSVS_VERSION` environment variable to the Visual Studio version (e.g. `2013` or `2015`)
 
 ## Instructions
 
 ```bash
-# Use the `Git Shell` program which was installed by GitHub Desktop.
-# Also make sure that you are logged into GitHub Desktop.
+# Use the `Git Shell` program which was installed by GitHub Desktop and make sure that you are logged into Github Desktop
```

Comment:
It's probably better to tell them to open the shell directly from the Github Desktop app in this case, which is a bit more streamlined than asking them to open the git shell first and then log into the Github Desktop app.